excel/src/main/java/tech/tablesaw/io/xlsx/XlsxReader.java

```diff
@@ -145,14 +138,16 @@ private Boolean isBlank(Cell cell) {
     return null;
   }
 
-  private ColumnType getColumnType(Cell cell) {
-    switch (cell.getCellType()) {
+  private ColumnType getColumnType(Cell cell, CellType cellType) {
+    switch (cellType) {
```

Comment:
I think I'd probably make the first line of this method get the cell type and leave this method with a single parameter

```
  private ColumnType getColumnType(Cell cell) {
      CellType cellType = cell.getCellType() == FORMULA ? cell.getCachedFormulaResultType() : cell.getCellType();
      switch (cellType) {
```

And do the same for `appendValue` below

Reply:
made the changes